vnext/Shared/OInstance.cpp

```diff
@@ -495,6 +468,12 @@ InstanceImpl::InstanceImpl(
       instanceArgs.EnableNativePerformanceNow = m_devSettings->enableNativePerformanceNow;
       instanceArgs.DebuggerConsoleRedirection = m_devSettings->debuggerConsoleRedirection;
 
+      // Disable bytecode caching with live reload as we don't make guarantees
+      // that the the bundle version will change with edits
+      if (m_devSettings->liveReloadCallback == nullptr) {
+        instanceArgs.ScriptMetadata = m_devSettings->chakraScriptMetadata;
```

Comment:
>chakraScriptMetadata [](start = 53, length = 20)

perhaps not part of this change, but for the future, it would be nice to move all of this chakra related functionality out of shared OInstance.cpp into its own file, like ChakraInstance.cpp

Reply:
I agree, though it looks like a lot of this should go away when we're on JSI only. The compile time flags for different engines to use for JSI above honestly bothers me the most. 

---
In reply to: [342751804](https://github.com/microsoft/react-native-windows/pull/3591#discussion_r342751804) [](ancestors = 342751804)